# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../ulogd2/mysql_header.patch
# Copyright (C) 2008 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

diff -Nur ulogd-2.0.0beta2-orig/output/mysql/ulogd_output_MYSQL.c ulogd-2.0.0beta2/output/mysql/ulogd_output_MYSQL.c
--- ulogd-2.0.0beta2-orig/output/mysql/ulogd_output_MYSQL.c	2008-08-27 11:04:23.000000000 +0200
+++ ulogd-2.0.0beta2/output/mysql/ulogd_output_MYSQL.c	2008-08-27 11:05:10.000000000 +0200
@@ -44,7 +44,7 @@
 #include <errno.h>
 #include <time.h>
 #include <arpa/inet.h>
-#include <mysql/mysql.h>
+#include <mysql.h>
 #include <ulogd/ulogd.h>
 #include <ulogd/conffile.h>
 #include <ulogd/db.h>
